src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -126,10 +116,23 @@ class npc_lady_sylvanas_windrunner : public CreatureScript
         uint32 BlackArrowTimer;
         uint32 ShotTimer;
         uint32 MultiShotTimer;
+        bool flag;
+        Player * _player;
```

Comment:
We never really store pointers to items/players/creatures as they can be invalid at some point, and that may lead to unexpected outcomes. You can just store it's guid instead of you need to access it out of the local method scope. I see you're checking it against null but it's still better to keep code consistent